src/core/udb-organizer.factory.js

```diff
@@ -48,7 +48,9 @@ function UdbOrganizerFactory(UitpasLabels) {
     parseJson: function (jsonOrganizer) {
       this['@id'] = jsonOrganizer['@id'];
       this.id = jsonOrganizer['@id'].split('/').pop();
-      this.name = jsonOrganizer.name || _.get(jsonOrganizer, 'name_deprecated', '');
+      this.name = _.get(jsonOrganizer.name, 'nl', null) ||
+        _.get(jsonOrganizer, 'name_deprecated', null) ||
```

Comment:
Is `name_deprecated` ondertussen nog nodig? Op https://search.uitdatabank.be/organizers/?embed=true&limit=10&start=0 krijg ik `name` terug, als `string` weliswaar.